src/base/server_type.hpp

```diff
@@ -24,7 +24,9 @@
 int init_server_type (const char *db_name);
 void finalize_server_type ();
 SERVER_TYPE get_server_type ();
-void set_server_type (SERVER_TYPE type);
+SERVER_TYPE get_value_from_config (server_type_config parameter_value);
 bool is_tran_server_with_remote_storage ();
+void set_server_type (SERVER_TYPE type);
+void setup_tran_server_params_on_single_node_config ();
```

Comment:
get_value_from_config & setup_tran_server_params_on_single_node_config may be declared in server_type.cpp; they don't have to be exposed with the header to other files.